src/core/coordConverter.js

```diff
@@ -0,0 +1,29 @@
+export class coordConverter {
+    constructor(scaleFactor) {
+        this.scaleFactor = scaleFactor;
```

Comment:
No problem at all but I just want to point out the inconsistency in the naming. 
it's called sf in main, scaleFactor here and then sf again right below